PuzzleSlider/src/ControllerTest/StopwatchTest.java

```diff
@@ -10,21 +10,72 @@
 
 public class StopwatchTest extends Stopwatch{
 	
+	private static final int DUMMY_DELAY = 99999;
+
 	@Test
 	public void elapsedTimeTest(){
 		Stopwatch stopwatch = new Stopwatch();
 		stopwatch.start(NumericalRepository.GAME_STOPWATCH_DELAY_MS, new PuzzleSlider());
 		stopwatch.pause();
 		stopwatch.resume();
 		
+		// where's da test? :D

```

Comment:
Runtime test -> if does not throw any exception in run. It seems redundant, but keep them in ( Gyarfas says never delete test :) )